unit_test/batched/Test_Batched_SerialGemm_Real.hpp

```diff
@@ -1,3 +1,30 @@
+#if defined(HAVE_KOKKOSKERNELS_HALFMATH)
```

Comment:
Note: This may run with `::Test::halfScalarType = float` if half precision is available on only the host or device.